src/games/strategy/triplea/delegate/BattleDelegateOrdered.java

```diff
@@ -0,0 +1,114 @@
+package games.strategy.triplea.delegate;
+
+import java.util.Collection;
+
+import games.strategy.engine.data.Territory;
+import games.strategy.triplea.delegate.BattleDelegate;
+import games.strategy.triplea.delegate.IBattle;
+import games.strategy.triplea.delegate.IBattle.BattleType;
+import games.strategy.util.Match;
+
+/**
+ * This is to support the new maps added to world_war_ii_global which enforce strategic bombing first in the combat order @MapSupport
```

Comment:
@MapSupport in javadoc comment does not do anything, should be applied above the class name.